docs/INSTALL.md

```diff
@@ -16,7 +16,7 @@ For _Windows_ you need to have [Git for Windows](https://git-scm.com/download/wi
 
 First of all get the source code. The full Organic Maps sources repository is ~8.5Gb in size, there are various [clone options](#special-cases-options) to reduce download size to suit your needs.
 
-For _Windows 10_ enable [symlinks](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks) support in git first:
+For _Windows 10_ enable [symlinks](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks) support in git first, if this doesn't enable symlinks add -c core.symlinks=true parameter to the clone command shown below:
```

Comment:
1. What about Windows 11?
2. How to check if it helped and enabled symlinks?

Reply:
Will try with Win11, I have tried it on 2 Win10 machines with the same results. I only knew that it didn't work was when I built it in AS and got those 2 language files errors (Discussion #3864)

Reply:
Again: these two language errors mean that symlinks in the git repo _were not enabled_. The main question is "why are they not enabled/not working?"

Reply:
Works fine on Win11 (followed the current released install doc as before) and when I run the b2 stuff it says:-
![Screenshot 2023-01-12 101757](https://user-images.githubusercontent.com/15140454/212086974-953cc19e-e675-4be2-9d62-6eab32b202a7.png)

where as on my Win10 pc's it says "symlinks supported : no (cached)"

Reply:
Does it mean that you did not enable symlinks in your Git installation? https://stackoverflow.com/a/52097145

Reply:
@stevegdavis can you please investigate what exactly git does if you set that checkmark "enable symlinks" during git installation? Does it set the same global config flag? Or does something else? It may be more reliable to do the same action as git installation does manually, so people won't need to reinstall the git again.

Can you please confirm that the original instruction did not work for you with that "symlinks" flag _enabled_ during the git installation?

Reply:
OK will do. Over the weekend I ran it all again on Win10, reinstalling Git for Windows but **without** checking the symlinks on install checkbox now if I run git config --global core.symlinks without the true param it actually returns true which seems strange, however  during the boost build step I can see messages something like hardlink and junction created but no symlinks created, that as expected fails the AS build with those language files. I will now try it again from scratch but this time checking the symlinks on install checkbox to see if that works, sorry it takes a few hours to do the full cycle as my Win10 machine is quite old.

Reply:
@biodranik - I feel that it would after all not necessary to alter the install doc at all. I have tried all possible options and found if at least 1 symlinks option is enabled it will work. Installing Git from scratch has the symlinks checkbox ticked but if a user unticks it or Git is already installed possibly without symlinks checkbox ticked then of course running the 'git config --global core.symlinks true' would be needed.
Running 'dir /AL /S c:\' from the command line will tell a user if symlinks are supported.
No other user has come forward with the same problem so I feel that changes are not needed.
Tests done on Win10 and 11.

Reply:
Non-working symlinks are now at the top of all requests, so it makes sense to clarify it in the details in the install.md.

Reply:
I noticed that this PR has been approved but it has not been merged maybe due to a DCO missing. I did originally create a PR but didn't do a DCO but then I closed it and opened another PR that I did add the DCO to so I don't know if that is the problem?